membrane_curvature/base.py

```diff
@@ -0,0 +1,68 @@
+import numpy as np
+
+from membrane_curvature.surface import get_z_surface
+from membrane_curvature.curvature import mean_curvature, gaussian_curvature
+
+
+from MDAnalysis.analysis.base import AnalysisBase
+import MDAnalysis as mda
+
+
+class MembraneCurvature(AnalysisBase):
+
+    def __init__(self, universe, select='all',
+                 n_x_bins=100, n_y_bins=100,
+                 x_range=None,
+                 y_range=None,
+                 pbc=True, **kwargs):
+
+        super().__init__(universe.universe.trajectory, **kwargs)
+        self.selection = universe.select_atoms(select)
+        self.pbc = pbc
+        self.n_x_bins = n_x_bins
+        self.n_y_bins = n_y_bins
+        self.x_range = x_range if x_range else (0, universe.dimensions[0])
+        self.y_range = y_range if y_range else (0, universe.dimensions[1])
+
+        # Raise if selection doesn't exist
+        if (self.selection.n_residues) == 0:
```

Comment:
Nothing wrong with this technically, but it might be good to simplify it to just `len(atomgroup) == 0` purely on the the basis that it's more descriptive of your intent / more widely used way of doing this?

Reply:
By renaming `selection` to `ag`, your suggestion reads better, so I updated it. Thanks.